### ISSUE TITLE
Build no longer seems able to find FDPP install directory

### ISSUE THREAD

**Issue by andrewbird** [dosemu2/dosemu2#2290]
**Describe the bug**
Don't seem to be able to build dosemu with fdpp support when fdpp is installed in a non standard location any more.

**To Reproduce**
1. Build fdpp with meson as follows (my pwd is /clients/common/fdpp.git)
~~~
INST="`pwd`/inst"                                                                                 
rm -rf build ${INST}                                                          
./configure.meson --prefix ${INST} build
meson compile --verbose -C build
meson install -C build
~~~
2. Build Dosemu2 as follows
~~~
FDPP_LIBRARY_SOURCE=/clients/common/fdpp.git
export PKG_CONFIG_PATH=${FDPP_LIBRARY_SOURCE}/inst/lib/pkgconfig                
./default-configure -d \                                                        
        --with-fdpp-lib-dir=${FDPP_LIBRARY_SOURCE}/inst/lib/fdpp \              
        --with-fdpp-kernel-dir=${FDPP_LIBRARY_SOURCE}/inst/share/fdpp \         
        --with-fdpp-include-dir=${FDPP_LIBRARY_SOURCE}/inst/include \           
        --with-fdpp-pkgconf-dir=${FDPP_LIBRARY_SOURCE}/inst/lib/pkgconfig       
~~~
Note FDPP is not detected when building plugin

**Attach the log**

cat /clients/common/dosemu2.git/src/plugin/fdpp/config.log
~~~
configure:1832: checking for pkg-config                                         
configure:1855: found /usr/bin/pkg-config                                       
configure:1867: result: /usr/bin/pkg-config                                     
configure:1892: checking pkg-config is at least version 0.9.0                   
configure:1895: result: yes                                                     
configure:2033: checking for FDPP                                               
configure:2040: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"               
Package fdpp was not found in the pkg-config search path.                       
Perhaps you should add the directory containing `fdpp.pc'                       
to the PKG_CONFIG_PATH environment variable                                     
Package 'fdpp', required by 'virtual:world', not found                          
configure:2043: $? = 1                                                          
configure:2057: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"               
Package fdpp was not found in the pkg-config search path.                       
Perhaps you should add the directory containing `fdpp.pc'                       
to the PKG_CONFIG_PATH environment variable                                     
Package 'fdpp', required by 'virtual:world', not found                          
configure:2060: $? = 1                                                          
configure:2074: result: no                                                      
Package 'fdpp', required by 'virtual:world', not found                          
configure:2090: error: fdpp installation not found or too old                   
~~~
~~~
ajb@calypso:/clients/common/dosemu2.git$ cat ../fdpp.git/inst/lib/pkgconfig/fdpp.pc
prefix=/clients/common/fdpp.git/inst
includedir=${prefix}/include
libdir=${prefix}/lib

Name: fdpp
Description: fdpp: fdpp
Version: 1.7
Libs: -L${prefix}/lib/fdpp -lfdpp -lfdldr
Cflags: -I${includedir}
~~~

I note that config.log lists `PKG_CONFIG_PATH='/usr/local/lib/pkgconfig'` as an output variable, is it ignoring my env setting?

**A regression?**
Yes, was okay in de18d6210c67ef, haven't checked later until now

**dosemu2 origins**
Build from source

**Additional info**
.

**Comment by stsp** [comment]
Is your dosemu git up-to-date?
I reproduced your steps, except for
doing configure directly in `plugins/fdpp`
dir for simplicity.
The log starts with:
```
configure:1832: checking for pkg-config
configure:1855: found /usr/bin/pkg-config
configure:1867: result: /usr/bin/pkg-config
configure:1892: checking pkg-config is at least version 0.9.0
configure:1895: result: yes
configure:1910: checking for /home/stas/src/fdpp/inst/lib/fdpp/libfdpp.so
configure:1925: result: yes
configure:1940: checking for /home/stas/src/fdpp/inst/include/fdpp/thunks.h
configure:1955: result: yes
configure:2033: checking for FDPP
configure:2040: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"
configure:2043: $? = 0
configure:2057: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"
configure:2060: $? = 0
configure:2098: result: yes
```

Please note that you don't even
have the checks for `libfdpp.so` and
`thunks.h`!

**Comment by andrewbird** [comment]
> Is your dosemu git up-to-date?

Yes, I pulled from upstream/devel and rebased my test changes on top.

Okay, I just did `(cd src && git clean -d -f -x)` and now it's fine. Weird, sorry for the noise!

Here's the new salient part
~~~
configure:1832: checking for pkg-config                                         
configure:1855: found /usr/bin/pkg-config                                       
configure:1867: result: /usr/bin/pkg-config                                     
configure:1892: checking pkg-config is at least version 0.9.0                   
configure:1895: result: yes                                                     
configure:1910: checking for /clients/common/fdpp.git/inst/lib/fdpp/libfdpp.so  
configure:1925: result: yes                                                     
configure:1940: checking for /clients/common/fdpp.git/inst/include/fdpp/thunks.h
configure:1955: result: yes                                                     
configure:2033: checking for FDPP                                               
configure:2040: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"               
configure:2043: $? = 0                                                          
configure:2057: $PKG_CONFIG --exists --print-errors "fdpp >= 1.7"               
configure:2060: $? = 0                                                          
configure:2098: result: yes                                                     
configure:2104: checking for VALGRIND                                           
configure:2111: $PKG_CONFIG --exists --print-errors "valgrind"                  
configure:2114: $? = 0                                                          
configure:2128: $PKG_CONFIG --exists --print-errors "valgrind"                  
configure:2131: $? = 0                                                          
configure:2171: result: yes                                                     
configure:2284: creating ./config.status                                        
~~~
